src/maga/character/Trump.java

```diff
@@ -14,25 +15,60 @@
  */
 public class Trump extends Character {
     /**
+     * Integer that sets how many rounds Trump should wait.
+     */
+    private int roundsToWait;
+    /**
      * The name of the character
      */
     public Trump() {
         super("Donald Trump");
     }
     /**
+     * This method is a get method for the rounds to wait.
+     * @return 
+     */
+    public int getRoundsToWait() {
+        return roundsToWait;
+    }
+    
+    /**
+     * This method sets the rounds to wait.
+     * @param roundsToWait
+     */
+    public void setRoundsToWait(int roundsToWait) {
+        this.roundsToWait = roundsToWait;
+    }
+    
+    /**
      * The method sets Trumps route to the steak item.
      * @param cleaningRoom is the room with the steak.
+     * @param player
      */
-    public void findSteak(Room cleaningRoom) {
-
-
+    public void findSteak(Room cleaningRoom, Player player) { 
+        if (player.getCurrentRoom() != cleaningRoom) {
+            System.out.println("You have to be in the cleaning room to use this method.");
+            return; 
+        }
+        if (player.hasItem("steak-with-ketchup")) {
+            this.setCurrentRoom(cleaningRoom);
+            roundsToWait = 3; 
+            player.dropItems(new Command(CommandWord.DROP, "steak-with-ketchup"));
+            player.goRoom(new Command(CommandWord.GO, "south"));
+            System.out.println("You called Trump to the cleaning room.");
+        }
```

Comment:
Would be nice to have a message if you don't have a steak with ketchup.
```java
if (!player.hasItem("steak-with-ketchup")) {
     System.out.println("You can't call trump without a steak with ketchup");
     return;
}

Reply:
fixed in: 871f79b749c3891c94e586cfcfecd9376b71ed2c